keyboards/vortex/m0110a/info.json

```
@@ -0,0 +1,492 @@
+{
+    "manufacturer": "Vortex",
+    "keyboard_name": "M0110A",
+    "maintainer": "Gsky",
+    "bootloader": "wb32-dfu",
+    "bootmagic": {
+        "matrix": [1, 6]
+    },
+    "diode_direction": "COL2ROW",
+    "dynamic_keymap": {
+        "layer_count": 5
+    },
+    "encoder": {
+        "enabled": false
+    },
+    "features": {
+        "audio": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
```

no encoder config specified
```suggestion
    "features": {
        "audio": false,
        "bootmagic": true,
        "command": false,
        "console": false,
```